### DISCUSSION TITLE
Is it possible to delete change request, or paste one change request in another document?

### DISCUSSION THREAD

**Discussion by petrope** [GitbookIO/community#470] [0. General bugs and questions] [answered]
Hi,
is it possible to delete change request, or paste one change request in another document?
thx
PP

**Comment by aikoii** [comment, marked as answer]
Hey @petrope! Currently, there is no option to delete a change request (CR), you can archive it
![CleanShot 2024-02-06 at 17 35 37@2x](https://github.com/GitbookIO/community/assets/106068994/29025679-6c1f-4fba-8061-31ace9c34a38)

As for your second question, to make similar changes to another document, you'll need to start a new CR for it and manually copy and paste the content you want from that other CR 🙂
